research/picai/fl_nnunet/nnunet_client.py

```diff
@@ -593,7 +639,7 @@ def get_properties(self, config: Config) -> Dict[str, Scalar]:
         # plans since client needs to be initialized to get properties
         config["nnunet_plans"] = plans_bytes
         properties = super().get_properties(config)
-        properties["nnunet_plans"] = plans_bytes
+        properties["nnunet_plans"] = pickle.dumps(plans_bytes)
```

Comment:
Maybe a clarifying comment here might be helpful. It's sort of funny that we're doing a `pickle.dumps(plans)`, which should return a binary object, and then we're doing a `pickle.dumps` again on that binary object.

Reply:
I swear I've deleted this change like 4 times and it keeps sneaking back in with the merges. We're not supposed to be double pickling the plans. I have no idea why the code still runs when this error is present.